bedrock/firefox/templates/firefox/developer/index.html

```diff
@@ -29,67 +29,68 @@
 {% block content %}
 {% include 'firefox/includes/hub/sub-nav.html' %}
 <main role="main">
-  <section class="section section-intro">
-    <div class="content">
+  <section class="t-intro">
+    <div class="mzp-l-content">
+      <h1 class="intro-title">{{ _('Firefox Quantum: Developer Edition') }}</h1>
+      <p class="intro-tagline">{{ _('Welcome to your new favorite browser. Get the latest features, fast performance, and the development tools you need to build for the open web.') }}</p>
 
-      <div class="section-body">
-        <h1 class="page-title">{{ _('Firefox Quantum: Developer Edition') }}</h1>
-        <p class="tagline">{{ _('Welcome to your new favorite browser. Get the latest features, fast performance, and the development tools you need to build for the open web.') }}</p>
-
-        {{ download_firefox('alpha', platform='desktop', dom_id='intro-download', button_color='button-arrow', download_location='primary cta') }}
-
-        <p class="feedback-note">
-          {{ _('Firefox Developer Edition automatically sends feedback to Mozilla.') }}
-          <a href="{{ url('privacy.notices.firefox') }}#telemetry" class="more">{{ _('Learn more') }}</a>
-        </p>
-      </div>
-
-      {{ high_res_img('firefox/developer/hero-screenshot.png', {'class': 'hero-image', 'alt': '', 'width': '1200', 'height': '400'}) }}
+      {{ download_firefox('alpha', platform='desktop', dom_id='intro-download', button_color='button-arrow', download_location='primary cta') }}
 
+      <p class="intro-feedback-note">
+        {{ _('Firefox Developer Edition automatically sends feedback to Mozilla.') }}
+        <a href="{{ url('privacy.notices.firefox') }}#telemetry" class="more">{{ _('Learn more') }}</a>
```

Comment:
The section this links to is closed in the updated privacy page. I'm not sure there's anything we can do about that within the scope of this PR but perhaps at least we can link to `#health-report` instead (that's the parent section containing the telemetry stuff).

Reply:
I updated it to http://localhost:8000/en-US/privacy/firefox/#pre-release which is the section that explains telemetry is collected differently in pre-release versions of Firefox.